.pre-commit-config.yaml

```diff
@@ -14,12 +13,20 @@ repos:
   - id: eslint
     name: eslint
     language: system
-    entry: sh -c 'npm run eslint'
+    entry: sh -c 'npm run eslint:fix'
     files: .js$
 - repo: local
   hooks:
   - id: stylelint
     name: stylelint
     language: system
-    entry: sh -c 'npm run stylelint'
+    entry: sh -c 'npm run stylelint:fix'
     files: .css$
+- repo: local
+  hooks:
+  - id: yapf
+    name: yapf
+    language: python
+    entry: yapf
+    files: \.py$
+    args: [-r, -e, "app/**/migrations/*.py", app/dataviz/, app/app/]
```

Comment:
Do we need to mention `*.py` here since we are already mentioning in the `files:` to look for only `.py` files?

Reply:
I mean, I can drop the `*.py` ending the exclusion.  It won't change anything.  I primarily left it like this to remain consistent with the makefile execution of `yapf`.